### PR TITLE
Add PyCon US 2023 branding

### PR DESCRIPTION
**Pull request by gustavwilliam**
Adds the following assets for PyCon US 2023.
> **Note**: The Banner requires the user to have the Rubik Mono One Regular and Kulim Park fonts installed to render properly. 

## Banner
pycon_banner

## Server Icon
pycon_us_icon

### CODE REVIEW

**Comment by gustavwilliam** [comment]
The 22nd of April is Earth Day, so we have a collision on that day. Do we want to *eradicate* Earth Day this year and let this event last until the 23rd, or do we simply let this event last until the 21st? Either is fine by me.

Also, @swfarnsworth, looking for a review of the event description in `meta.md`.

**Comment by swfarnsworth** [comment]
Thank you, these look amazing!

I don't think the text in the SVG version of the banner is supposed to look like this?

image

**Comment by swfarnsworth** [comment]
Sorry, I forgot to address the collision with Earth Day. I think PyCon definitely takes precedence, so unless we're fine with having Earth Day branding only on days that aren't Earth Day, we should simply not have Earth Day branding this year.

File: events/pycon_us/meta.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+start_date: April 19
+end_date: April 23
+---
+**PyCon US**
+
+Celebrating the 20th Anniversary of PyCon US, a Python conferance that hosts tutorials, talks, a job fair, and sprints! To learn more about it, go to [us.pycon.com](https://us.pycon.org/2023/).
```
Comment (dawnofmidnight) [inline events/pycon_us/meta.md:7]:
```suggestion
Celebrating the 20th Anniversary of PyCon US, a Python conference that hosts tutorials, talks, a job fair, and sprints! To learn more about it, go to [us.pycon.com](https://us.pycon.org/2023/).
```

File: events/pycon_us/meta.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+start_date: April 19
+end_date: April 23
+---
+**PyCon US**
+
+Celebrating the 20th Anniversary of PyCon US, a Python conference that hosts tutorials, talks, a job fair, and sprints! To learn more about it, go to [us.pycon.com](https://us.pycon.org/2023/).
```
Comment (dawnofmidnight) [inline events/pycon_us/meta.md:7]:
```suggestion
Celebrating the 20th Anniversary of PyCon US, a Python conference that hosts tutorials, talks, a job fair, and sprints! To learn more about it, go to [us.pycon.org](https://us.pycon.org/2023/).
```

**Comment by gustavwilliam** [comment]
I don't think this PR is relevant anymore, let's close.

**Pull request closed without merging**